app/controllers/reminders/edit.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    updateReminder() {
+      this.transitionToRoute('reminders')
```

Comment:
So in ember, records can be modified at anytime but those modifications will be considered 'dirtyAttributes' till the model is saved. This allows people to 'rollbackAttributes' to the previous state.